text/0022-versions.md

```diff
@@ -0,0 +1,60 @@
+# Contributing guidelines
+- Start Date: 2020/02/06
+- RFC PR: https://github.com/openmrs/openmrs-rfc-frontend/pull/22
+
+## Decision, including impact on distributions
+OpenMRS frontend ESMs should be versioned according to
+[Semantic Versioning](https://semver.org/), with the following modification
+to the definition of a major version:
+
+A **Major** version is one in which **any** of the following have taken place
+- Backward-incompatible changes have been made to the API, if the ESM exposes an API
+- Backward-incompatible changes have been made to the Configuration API,
+  such that upgrading might cause a previously valid configuration to be invalid
+- A functional and substantive UI change has been introduced and that change is
+  present in the default configuration
+- A backend module dependency has been introduced or upgraded across a minor version
+
```

Comment:
Can we also include:

> A frontend module dependency has been introduced or upgraded across a minor version

Reply:
Can you provide an example of when that would happen?

Reply:
Nevermind, handled by semver.